profiles/rawls-service-perimeters/terraform/rawls-service-perimeters/create-sam-perimeter-resource.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+
+# This script is run by terraform to create a service perimeter in Sam.
```

Comment:
Can we add some more comments on what the script does, and what terraform-y things it doesn't do?